### PR TITLE
Ensure that OL8 `/boot` partition is a multiple of the RHEL8 partition-size

### LINKED ISSUE #675: [BUG] The `/boot`  partition is too small for `UEK`-enabled Oracle Linux 8 AMIs
### Expected behavior

One _should_ be able to install kernel updates as part of a `dnf update` operation.

### Actual behavior

Due to Oracle Linux 8 having a _much_ fatter kernel than RHEL 8 or CentOS Stream 8, the `/boot`  partition on Oracle Linux 8 images ends up being too small when using the default partition-sizes. This prevents any `dnf update` operations that involve updating the `kernel` RPMs.

### Steps to reproduce behavior

1. Launch a system from an Oracle Linux 8 AMI that is a couple months old
2. Attempt to patch up the resulting EC2 (e.g., `dnf update -y`)
3. Operation fails due to insufficient space in `/boot`
 

### Suggestions for fix

Either:
* Update the `amigen8-build.sh` script to pass a "suitable for all distros" size-override flag-option (`-b`/`--bootprt-size`) to the AMIgen8 `DiskSetup.sh` script
* Update the `amigen8-build.sh` script to pass a "suitable for Oracle Linux 8" size-override flag-option (`-b`/`--bootprt-size`) to the AMIgen8 `DiskSetup.sh` script for _just_ OL8 builds

### PR DESCRIPTION
**Pull request by ferricoxide**
Fixes #675 

* New PR Alert to: @plus3it/spel

### CODE REVIEW

File: spel/scripts/amigen8-build.sh [hunk 1]
```diff
@@ -394,6 +396,21 @@ function ComposeDiskSetupString {
         DISKSETUPCMD+="-B ${AMIGENBOOTSIZE} "
     fi
 
+    # Set the size of the /boot filesystem
+    if  [[ -f /etc/oracle-release ]] ||
```
Comment (lorengordon) [inline spel/scripts/amigen8-build.sh:400]: You can actually just check the `BUILDER` env if you want... See the `case` statement on line 77...

```
BUILDER=ol-8
```

Comment (ferricoxide) [reply spel/scripts/amigen8-build.sh:400]: Ok. Looks like I didn't scope-limit `BUILDER`, so I'll see if I can leverage _that_.

Comment (lorengordon) [reply spel/scripts/amigen8-build.sh]: Yeah, `BUILDER` is meant to be available essentially as a "global", throughout the whole script...

Comment (ferricoxide) [reply spel/scripts/amigen8-build.sh]: Ok. Switching to that still resulted in a good build. Marking this thread resolved.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**